shared/src/commonMain/kotlin/com/getbux/simplebux/kmm/shared/BuxAPI.kt

```diff
@@ -18,7 +18,7 @@ import kotlinx.serialization.json.Json
 data class Money(
     val currency: String,
     val decimals: Int,
-    private var amount: Double
+    var amount: Double
```

Comment:
Any idea why this needs to be a var?

Reply:
I wanted to display the amount, so that is why I removed the private keyword. I don't think it needs to be a var, it's more just muscle memory.

Reply:
Shouldn't be a var, don't remember why I had it like that in the first place. I will change it later in another PR.